config/env/test/database.js

```diff
@@ -2,7 +2,7 @@ module.exports = ({ env }) => ({
     connection: {
       client: 'sqlite',
       connection: {
-        filename: env('DATABASE_FILENAME', 'integration-test/data.db'),
+        filename: env('DATABASE_FILENAME', process.env.PLAYWRIGHT === 'true' ? '.tmp/test.db' : 'integration-test/data.db'),
```

Comment:
Do we need to set `PLAYWRIGHT=true` in the github workflow env?

Reply:
It's set here https://github.com/dev-launchers/strapiv4/pull/163/files#diff-f679bf1e58e8dddfc6cff0fa37c8e755c8d2cfc9e6b5dc5520a5800beba59a92R17 and will be removed after I clean up current postman tests.